client.go

```diff
@@ -193,7 +193,7 @@ func runCommandInContainer(img string, cmd []string, streaming bool) (string, er
 // returned can be of type dockerError. This only happens when the container
 // has run normally (no signals, no timeout), but the exit code is not 0. Read
 // the documentation on the `dockerError` command on why we do this.
-func startContainer(img string, cmd []string, streaming, wait bool) (string, error) {
+func startContainer(img string, cmd []string, streaming, wait bool, dst io.Writer) (string, error) {
```

Comment:
Since `dst` and `streaming` are intertwined options, why not just have `dst`? If we have `dst` that implies that we're streaming.